cluster/addons/fluentd-elasticsearch/fluentd-es-configmap.yaml

```diff
@@ -101,25 +111,19 @@ data:
     # CRI Log Example:
     # 2016-02-17T00:04:05.931087621Z stdout F [info:2016-02-16T16:04:05.930-08:00] Some log text here
     <source>
-      type tail
+      @id fluentd-containers.log
+      @type tail
       path /var/log/containers/*.log
       pos_file /var/log/es-containers.log.pos
       time_format %Y-%m-%dT%H:%M:%S.%NZ
       tag raw.kubernetes.*
+      format json
       read_from_head true
```

Comment:
Why do we remove the `multi_format`? Without this elasticsearch-fluentd won't work with containerd or cri-o.